packages/donor/src/components/DonationInfo/DonationInfo.tsx

```diff
@@ -1,24 +1,27 @@
-import styles from "./DonationToBookInfo.module.scss";
+import styles from "./DonationInfo.module.scss";
 import Calendar from "../../assets/images/AppointmentCalendar.svg";
 import { DateUtils, Hospital, LocaleUtils } from "@zm-blood-components/common";
-import React from "react";
 
 export interface DonationToBookInfoProps {
   donationStartTimeMillis: number;
   hospital: Hospital;
+  isPreviousAppointmentInfo: boolean;
 }
 
 export default function DonationToBookInfo({
```

Comment:
If you rename the file, please rename the function and the props.

Reply:
Done